interface/billing/edih_view.php

```diff
@@ -33,7 +33,7 @@
     <meta http-equiv="content-type" content="text/html;charset=utf-8" />
     
     <!-- TODO: Address no_bootstrap here !-->
-    <?php Header::setupHeader(['no_main-theme', 'no_bootstrap', 'datetime-picker', 'datatables', 'datatables-jqui', 'datatables-jqui-theme', 'datatables-scroller', 'datatables-scroller-jqui-theme', 'jquery-ui', 'jquery-ui-sunny']); ?>
+    <?php Header::setupHeader(['no_main-theme', 'datetime-picker', 'datatables', 'datatables-jqui', 'datatables-jqui-theme', 'datatables-scroller', 'datatables-scroller-jqui-theme', 'jquery-ui', 'jquery-ui-sunny']); ?>
```

Comment:
regarding the datatables-jqui and datatables-jqui-theme , this substitution worked well elsewhere:
https://github.com/openemr/openemr/pull/3048/files

for scroller, guessing may need to substitute the datatables-scroller-jqui-theme (datatables.net-scroller-jqui) with https://www.npmjs.com/package/datatables.net-scroller-bs4

Reply:
Thanks!, i was just thinking what alternative i have.